keyboards/keychron/v4/iso/keymaps/mischback/rules.mk

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# SPDX-FileCopyrightText: 2023 Mischback <mischback.dev@googlemail.com>
+# SPDX-FileType: SOURCE
+
+# The Keychron V4 features a DIP switch at the back that let the user switch
+# between Mac and Windows layouts. As I run Linux anyways, this switch is not
+# in use for me personally.
+#
+# While this feature is deactivated, the default selection of layers in
+# Keychron's default keymap is not working and the first layer will be applied
+# as the default to the keyboard.
+#
+# TODO: This might be used for something different...
+#       See https://github.com/qmk/qmk_firmware/blob/master/docs/feature_dip_switch.md
+#       THIS WILL ALSO ENABLE Keychron's default function for switching layers
+#       depending on the position of the DIP, but that function is meant to be
+#       overridden by keymap specific code (see ``keyboards/keychron/v4/v4.c``),
+DIP_SWITCH_ENABLE = no
```

Comment:
you can also use this to disable the functionality too:
```c
bool dip_switch_update_user(uint8_t index, bool active) {
     return false;
}
```
You could also use it as a toggle switch for enabling NKRO, or other features, if you wanted. 

That said, at a bare minimum links shouldn't be present. 
```suggestion
# The Keychron V4 features a DIP switch at the back that let the user switch
# between Mac and Windows layouts. As I run Linux anyways, this switch is not
# in use for me personally.
#
# While this feature is deactivated, the default selection of layers in
# Keychron's default keymap is not working and the first layer will be applied
# as the default to the keyboard.
#
# TODO: This might be used for something different...
#       THIS WILL ALSO ENABLE Keychron's default function for switching layers
#       depending on the position of the DIP, but that function is meant to be
#       overridden by keymap specific code (see ``keyboards/keychron/v4/v4.c``),
DIP_SWITCH_ENABLE = no
```

Reply:
Thank you for this hint. I assume you will have noticed that I currently just don't use the DIP switch at all, so the deactivation is intended. I adjusted the comment in 6d2239792eec9777fd88c81058a7071791813cf9.

Regarding links in the inline comments: May be a matter of coding style. I can see why hardcoded links may be undesired and removed them.